blockchain/blockchain.go

```diff
@@ -188,9 +188,15 @@ func NewBlockChain(db database.DBManager, cacheConfig *CacheConfig, chainConfig
 			CacheSize:      512,
 			BlockInterval:  DefaultBlockInterval,
 			TriesInMemory:  DefaultTriesInMemory,
-			TrieCacheLimit: 0,
+			TrieNodeCacheConfig: statedb.TrieNodeCacheConfig{
+				CacheType:          statedb.CacheTypeFast,
+				FastCacheSizeMB:    0,
+				RedisEndpoints:     nil,
+				RedisClusterEnable: false,
+			},
```

Comment:
I see some places that uses these values. What do you think of creating functions such as `GetEmptyCacheConfig()`?

Reply:
The suggestion applied. Thanks!